File: DynamicKey/AgoraDynamicKey/php/sample/FpaTokenBuilderSample.php
<?php
require_once("../src/FpaTokenBuilder.php");

// Need to set environment variable AGORA_APP_ID
$appId = getenv("AGORA_APP_ID");
// Need to set environment variable AGORA_APP_CERTIFICATE
$appCertificate = getenv("AGORA_APP_CERTIFICATE");

$token = FpaTokenBuilder::buildToken($appId, $appCertificate);
echo 'Token with FPA service: ' . $token . PHP_EOL;

File: DynamicKey/AgoraDynamicKey/ruby/sample/fpa_token_builder_sample.rb
require_relative '../lib/dynamic_key2'

# Need to set environment variable AGORA_APP_ID
app_id = ENV['AGORA_APP_ID']
# Need to set environment variable AGORA_APP_CERTIFICATE
app_certificate = ENV['AGORA_APP_CERTIFICATE']

token = AgoraDynamicKey2::FpaTokenBuilder.build_token(app_id, app_certificate)
puts "Token with FPA service: #{token}"
